.circleci/config.yml

```diff
@@ -49,6 +49,10 @@ commands:
           name: Valgrind
           command: bash -l -c "make valgrind"
 
+      - run:
+          name: Benchmark
+          command: bash -l -c "make benchmark"
+
```

Comment:
CI for sanity?

Reply:
Yes, and also for regression (we can compare the results before and after introducing new features)